py-polars/polars/dataframe/frame.py

```diff
@@ -3387,15 +3393,15 @@ def write_ipc_stream(
         file: str | Path | IO[bytes],
         *,
         compression: IpcCompression = "uncompressed",
-        future: bool | None = None,
+        future: bool | int | None = None,
     ) -> None: ...
 
     def write_ipc_stream(
         self,
         file: str | Path | IO[bytes] | None,
         *,
         compression: IpcCompression = "uncompressed",
-        future: bool | None = None,
+        future: bool | int | None = None,
```

Comment:
`bool | int | None` seems like a weird signature to me. I think we must come up with a better API.

Reply:
Unfortunately we can do little giving Python polars 1.0 is already released. Changing the signature can only be done in a backward compatible way.

Reply:
`unstable` means we do not need to be backward compatible, so we're free to change the API here.

Reply:
Then I think we can use an enum.